tools/elf2hunk.c

```diff
@@ -541,11 +585,13 @@ static int relocate
                 break;
 
             case SHN_COMMON:
-                bug("[ELF2HUNK] SHN_COMMON symbol '%s' unsupported\n", symname);
-                      set_error(EINVAL);
-		      
-                return 0;
+                hunk = bssIndex;
+                value = theBSS->size;
+                theBSS->size += (sym.size + 3) & ~3;
 
+                symptr->shindex = htons(bssIndex);
+                symptr->value = htonl(value);
+            break;
```

Comment:
There're couple of formatting issues here and there. Please go over the PR carefully and fix them.